lib/statsample-glm/glm/base.rb

```diff
@@ -49,6 +63,23 @@ def coefficients as_a=:vector
         end
       end
 
+      # Returns the standard error obtained using the model
```

Comment:
Rather say "Returns the standard errors for the coefficient estimates".
It looks to me like it returns a vector of standard errors, containing an individual standard error for each coefficient.